### ISSUE TITLE
findNestedDependencies not correct in windows

### ISSUE THREAD

**Issue by cntanglijun** [jlouns/gulp-requirejs-optimize#13]
### My work dir tree like this:

![default](https://cloud.githubusercontent.com/assets/13279117/14467791/7737e5aa-010f-11e6-8821-82dfd7856355.png)
### src/module/a.js

``` js
define(function (require, exports, module) {
  console.log('module A');
});
```
### src/module/b.js

``` js
define(function (require, exports, module) {
  console.log('module B');
});
```
### src/sub-dir/main.js

``` js
require(['../requirejs-dev-config'], function () {
  require([
    '../module/a',
    '../module/b'
  ]);
});
```
### src/main.js

``` js
require(['./requirejs-dev-config'], function () {
  require([
    './module/a',
    './module/b'
  ]);
});
```
### src/requirejs-dev-config.js

``` js
// requirejs config file
// http://requirejs.org/docs/api.html#config
require.config({
  // clear browser's cache
  urlArgs: "bust=" + (new Date()).getTime(),

  baseUrl: '.',

  paths: {
    // http://jquery.com/
    jquery: '//cdn.bootcss.com/jquery/2.2.1/jquery.min',

    // http://jashkenas.github.io/underscore/
    underscore: '//cdn.bootcss.com/underscore.js/1.8.3/underscore-min'
  }
});
```
### gulpfile.js

``` js
'use strict';

const gulp = require('gulp');
const gulpRequrejsOptimize = require('gulp-requirejs-optimize');

gulp.task('dist', () => {
  gulp.src([
      'src/**/main.js'
    ])
    .pipe(gulpRequrejsOptimize({
      baseUrl: 'src',
      optimize: 'none',
      findNestedDependencies: true,
    }))
    .pipe(gulp.dest('dist'));
});
```
### dist/main.js

``` js
// requirejs config file
// http://requirejs.org/docs/api.html#config
require.config({
  // clear browser's cache
  urlArgs: "bust=" + (new Date()).getTime(),

  baseUrl: '.',

  paths: {
    // http://jquery.com/
    jquery: '//cdn.bootcss.com/jquery/2.2.1/jquery.min',

    // http://jashkenas.github.io/underscore/
    underscore: '//cdn.bootcss.com/underscore.js/1.8.3/underscore-min'
  }
});
define("requirejs-dev-config", function(){});

define('module/a',['require','exports','module'],function (require, exports, module) {
  console.log('module A');
});
define('module/b',['require','exports','module'],function (require, exports, module) {
  console.log('module B');
});
require(['./requirejs-dev-config'], function () {
  require([
    './module/a',
    './module/b'
  ]);
});
define("main.js", function(){});
```
### dist/sub-dir/main.js

``` js
require(['../requirejs-dev-config'], function () {
  require([
    '../module/a',
    '../module/b'
  ]);
});
define("sub-dir\main.js", function(){});
```

**The dist/sub-dir/main.js is not include the nested dependencies. In mac os, it's right.But in windows, it's incorrect.**

**Comment by jlouns** [comment]
Could you try this again with v1.0.0? There were some issues with windows paths in the last version.

**Comment by cntanglijun** [comment]
The problem is fixed in v1.0.0, thank you very much.:sparkles::+1:
